rfc-connector/README.md

```diff
@@ -13,10 +13,11 @@ So the Connector needs to be deployed as a `.war` Java application on Cloud Foun
 
 ## Development
 
-(optional) compile in a version of JCo for local dev: `mvn install:install-file -Dfile=sapjco3.jar -DgroupId=com.sap.conn.jco -DartifactId=com.sap.conn.jco.sapjco3 -Dversion=3.1.10 -Dpackaging=jar`
+(optional) compile in a version of JCo for local dev:
+`mvn install:install-file -Dfile=sapjco3.jar -DgroupId=com.sap.conn.jco -DartifactId=com.sap.conn.jco.sapjco3 -Dversion=3.1.10 -Dpackaging=jar`
```

Comment:
@vringar can we move the two sapjco3 files into a `lib` folder in the connector directory?

Reply:
Follow-up in #91